meson.build

```diff
@@ -144,8 +144,6 @@ if cc.get_id() == 'clang'
     assert(get_option('c_std') == 'gnu11', 'Remove this if HSE moves to c11 instead of gnu11')
     add_project_arguments(
         cc.get_supported_arguments([
```

Comment:
If you were able to get rid of this, please remove the entire enclosing if block.

Reply:
Done.